snippets/gift-card-recipient-form.liquid

```diff
@@ -0,0 +1,164 @@
+{% comment %}
+  Renders gift card recipient form.
+  Accepts:
+  - product: {Object} product object.
+  - form: {Object} the product form object.
+  - section: {Object} section to which this snippet belongs.
+
+  Usage:
+  {% render 'gift-card-recipient-form', product: product, form: form, section: section %}
+{% endcomment %}
+
+<div class="customer">
```

Comment:
Oh I think that's something that needed tweaking. Because some styling applied on the customer pages matched what was applied here, I think this class was used but should be duplicated with a more accurate naming if it's still useful or remove if it's not necessary